lib/i18n/tasks/translators/deepl_translator.rb

```diff
@@ -17,7 +17,10 @@ def initialize(*)
     protected
 
     def translate_values(list, from:, to:, **options)
-      DeepL.translate(list, to_deepl_compatible_locale(from), to_deepl_compatible_locale(to), options).map(&:text)
+      output = DeepL.translate(list, to_deepl_compatible_locale(from), to_deepl_compatible_locale(to), options)
+      return output.map(&:text) unless output.is_a? DeepL::Resources::Text
```

Comment:
Stylystically, I think this would be a bit more readable:

```ruby
if output.is_a?(DeepL::Resources::Text)
  output.text
else
  output.map(&:text)
end
```